lib/SystemSettingsConfig.php

```diff
@@ -236,4 +239,16 @@ private function checkFallbackLanguageLoop(string $source, array $definitions, a
             throw new \Exception("Language `$source` doesn't exist");
         }
     }
+
+    private static function getValuesFromContainerConfig():array {
```

Comment:
```suggestion
    private static function getConfigValuesFromContainer():array {
```

Reply:
Done in [d2af5ef](https://github.com/pimcore/pimcore/pull/15822/commits/d2af5efd63399a6e51db7ee91c0a38c04765d1b6)